evm_loader/lib/src/commands/transaction_executor.rs

```diff
@@ -45,21 +46,21 @@ impl Stats {
         self.created_objects += 1;
     }
 }
-pub struct TransactionExecutor {
-    pub client: Arc<dyn rpc::Rpc>,
+pub struct TransactionExecutor<'a, 'b> {
+    pub client: &'a dyn rpc::Rpc,
     pub send_trx: bool,
     pub signatures: RwLock<Vec<Signature>>,
-    pub stats: RwLock<Stats>,
-    pub fee_payer: Arc<dyn Signer>,
+    pub stats: RefCell<Stats>,
```

Comment:
Do you need `RwLock` for signatures here?

Reply:
I replaced it with a `RefCell`.